tv_apps/smart-system/js/app_modal_dialog.js

```diff
@@ -63,6 +63,14 @@
   };
 
   AppModalDialog.prototype._fetchElements = function amd__fetchElements() {
+    // XXX: Place another container outside of the original element to prevent
```

Comment:
Please file a bug for refactoring smart-modal-dialog to support DOM element and put the bug here.